src/pages/index.js

```diff
@@ -0,0 +1,331 @@
+import "./index.css";
+import { settings } from "../scripts/validation.js";
+import { disabledButton } from "../scripts/validation.js";
+import { resetValidation } from "../scripts/validation.js";
+import { enableValidation } from "../scripts/validation.js";
+import avatarSrc from "../images/avatar.jpg";
+import Api from "../utils/api.js";
+import { setButtonText } from "../utils/helpers.js";
+
+const avatarImage = document.getElementById("image-avatar");
+avatarImage.src = avatarSrc;
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "bd0dda27-19f5-4140-81e1-0cd5551e39ec",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, users]) => {
+    console.log(cards, users);
+    console.log("User data:", users); // Let's keep this to see what we get
+    currentUserId = users._id;
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardsList.prepend(cardElement);
+    });
+    const profileImage = document.querySelector("#image-avatar");
+    profileImage.src = users.avatar;
+
+    const profileTitle = document.querySelector(".profile__title");
```

Comment:
` document.querySelector(".profile__title");`

You've already found the profile name in line 45 as `profileName`. Please, reuse it everywhere